### ISSUE TITLE
Watch for "remains the same" in code returned from the LLM and reject it outright

### ISSUE THREAD

**Issue by TheTiGuR** [stackblitz/bolt.new#629]
When requesting specific changes, sometimes the response is extra literal (I think this happens from safeguards you put in place). See Screenshot: 
![CleanShot 2024-10-15 at 22 53 11](https://github.com/user-attachments/assets/568182fe-2242-4402-be19-80aeafb07565)

So, basically, if you see the response comes back with "...remains the same" in it, and it wasn't there in what was sent to the LLM, it should either be auto-rejected, or presented for approval before applying it to the existing files.

**Comment by profdl** [comment]
Yes! This is a big problem. Particularly as the project get larger.
